src/SearchNames.tsx

```diff
@@ -0,0 +1,64 @@
+import './MovieList.css'
```

Comment:
You don't need to import 'MovieList.css' here, because you've already imported it in 'MovieList.tsx' file.